### ISSUE TITLE
Results not so stable in SVHN to MNIST

### ISSUE THREAD

**Issue by RsEnts** [erictzeng/adda#2]
Hello, I am trying to rerun the code from SVHN to MNIST, however, I find that the results are not so robust when I try to rerun the code for several times using the default settings. The results(accuracy) seems to fluctuate from 0.68 to 0.79. Are there any suggestions about how to make it stable?

**Comment by jhoffman** [comment]
I have found that pre-training the discriminator adds stability. Also, training with a lower learning rate for longer.

**Comment by llf1234** [comment]
hello i also find the unstable problem again，but how to pretrain the discriminator？thank you
